lsd: add page (#6654)
tldr-pages/tldr: pages/common/lsd.md

# lsd

> List directory contents.

> > More information: <https://github.com/Peltoche/lsd>.

Reviewer: Adding this let's the user know the difference between this and `ls`.
```suggestion
> The next generation `ls` command, written in Rust.
> More information: <https://github.com/Peltoche/lsd>.
```

> - List files one per line with many details:

Reviewer 2: ```suggestion
- List files and directories, one per line.
```

`lsd -1`

> - List all files, including hidden files:

Reviewer 2: ```suggestion
- List all files and directories, including hidden ones, in the current directory:
```

Reviewer: I think it can be reduced to
```suggestion
- List all files and directories, including hidden ones:
```

`lsd -a`

> - List all files, with trailing `/` added to directory names:

Reviewer 2: ```suggestion
- List all files and directories, with trailing `/` added to directory names:
```

> - List all files and directories, with trailing `/` added to directory names:

Reviewer: ```suggestion
- List all files and directories with trailing `/` added to directory names:
```

`lsd -F`

> - Long format list (permissions, ownership, size, and modification date) of all files:

Reviewer: ```suggestion
- List all files and directories in long format (permissions, ownership, size, and modification date):
```

`lsd -la`

> - Long format list with size displayed using human-readable units (KiB, MiB, GiB):

Reviewer: ```suggestion
- List all files and directories in long format with size displayed using human-readable units (KiB, MiB, GiB):
```

`lsd -lh`

> - Long format list sorted by size (descending):

Reviewer: ```suggestion
- List all files and directories in long format, sorted by size (descending):
```

`lsd -lS`

> - Long format list of all files, sorted by modification date (oldest first):

Reviewer: ```suggestion
- List all files and directories in long format, sorted by modification date (oldest first):
```
